scripts/metricsdocgen/metrics

```diff
@@ -0,0 +1,576 @@
+# HELP coderd_api_active_users_duration_hour The number of users that have been active within the last hour.
```

Comment:
This will go out of date pretty quickly I think. Can you not just import the packages with `_ "blah"` and access the global prometheus registry directly to get this information?

Reply:
I don't think that this is doable without hacking. Here are all the places to import: https://github.com/coder/coder/issues/3520#issuecomment-1328899206. What is your recommendation for these entries?

I wouldn't like to create an artificial structure and have a dependency on code snippets using metrics. In this case, the only possible way is to have a global structure `CoderMetrics` which keeps track of all metrics.

Reply:
Yikes you're right, that's a lot. I think this is fine for now then 👍